performance/src/api_performance_tests.ts

```diff
@@ -174,10 +173,14 @@ const getloadTests = (
       ),
       !!sD?.workspace.projectId,
     ),
-    test(
-      "get project columns",
-      getRequest(`/api/v1/projects/${sD?.workspace.projectId}/columns`),
-    ),
+    // This is a bad endpoint and we know it is bad.
+    // No sense in making other endpoints be slowed before we get the fix in
+    // https://hpe-aiatscale.atlassian.net/browse/DET-10114
```

Comment:
Can we ensure that a ticket is created or a comment is made in one of these tickets to re-enable this test. Although I am assume that was the plan in the ticket anyway, I just don't want us to forget.